<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FramelessWidget</class>
 <widget class="QWidget" name="FramelessWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FramelessWidget</string>
  </property>
  <widget class="QWidget" name="root_widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>6</y>
     <width>665</width>
     <height>475</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.root {
    background-color: rgb(255, 255, 255);
    color: rgb(0, 0, 0);
    border: 1px solid rgb(255, 255, 255);
    border-radius: 6px;
}</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>32</y>
      <width>653</width>
      <height>343</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="title_widget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>32</height>
        </size>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>651</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QToolButton" name="toolButton_icon">
           <property name="styleSheet">
            <string notr="true">border: 0px</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_title">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>WindowTitle</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_min">
           <property name="toolTip">
            <string>最小化</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::ListRemove"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_max">
           <property name="toolTip">
            <string>最大化/还原</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::ListAdd"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_close">
           <property name="toolTip">
            <string>关闭</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="empty_widget" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
